backend/manager/modules/utils/src/main/java/org/ovirt/engine/core/utils/ovf/OvfWriter.java

```diff
@@ -609,6 +611,10 @@ public String getStringRepresentation() {
     protected abstract void startDiskSection();
     protected abstract void startVirtualSystem();
 
+    protected void writeNuma() {
+        // Do nothing.
+    }
```

Comment:
because then it will be outside the hardware section, unless i copy the whole parent method. i don't have a strong opinion whether leaving it as is, or override writeHardware (which leave it outside the hardware section), or just copy all of it?

Reply:
doesn't seem so, moved back to general section.